airdrop/airdrop-distributor/tests/functions/claim.rs

```diff
@@ -0,0 +1,356 @@
+
+        assert_eq!(
+            wallet1
+                .wallet
+                .get_asset_balance(&AssetId::new(*asset.asset_id))
+                .await
+                .unwrap(),
+            0
+        );
+        assert_eq!(
+            claim_data(&deploy_wallet.airdrop_distributor, identity_a.clone())
+                .await
+                .claimed,
+            false
+        );
+
+        claim(
+            airdrop_leaves[0].1,
+            asset.asset_id,
+            &deploy_wallet.airdrop_distributor,
+            key,
+            num_leaves,
+            proof.clone(),
+            identity_a.clone(),
+        )
+        .await;
+
+        assert_eq!(
+            wallet1
+                .wallet
+                .get_asset_balance(&AssetId::new(*asset.asset_id))
+                .await
+                .unwrap(),
+            airdrop_leaves[0].1
+        );
+        assert_eq!(
+            claim_data(&deploy_wallet.airdrop_distributor, identity_a.clone())
+                .await
+                .claimed,
+            true
+        );
+    }
+
+    #[tokio::test]
+    async fn claims_manual_tree() {
+        let (deploy_wallet, wallet1, wallet2, wallet3, asset) = setup().await;
+        let (identity_a, _, _, minter, key, num_leaves, asset_supply, airdrop_leaves, claim_time) =
+            defaults(&deploy_wallet, &wallet1, &wallet2, &wallet3).await;
+
+        let (root, proof1, proof2) = build_tree_manual(airdrop_leaves.clone()).await;
+
+        airdrop_constructor(
+            asset.asset_id,
+            claim_time,
+            &deploy_wallet.airdrop_distributor,
+            root,
+        )
+        .await;
+        asset_constructor(asset_supply, &asset.asset, minter).await;
+
+        assert_eq!(
+            wallet1
+                .wallet
+                .get_asset_balance(&AssetId::new(*asset.asset_id))
+                .await
+                .unwrap(),
+            0
+        );
+        assert_eq!(
+            claim_data(&deploy_wallet.airdrop_distributor, identity_a.clone())
+                .await
+                .claimed,
+            false
+        );
+
+        claim(
+            airdrop_leaves[0].1,
+            asset.asset_id,
+            &deploy_wallet.airdrop_distributor,
+            key,
+            num_leaves,
+            [proof1, proof2].to_vec(),
+            identity_a.clone(),
+        )
+        .await;
+
+        assert_eq!(
+            wallet1
+                .wallet
+                .get_asset_balance(&AssetId::new(*asset.asset_id))
+                .await
+                .unwrap(),
+            airdrop_leaves[0].1
+        );
+        assert_eq!(
+            claim_data(&deploy_wallet.airdrop_distributor, identity_a.clone())
+                .await
+                .claimed,
+            true
+        );
+    }
+}
+
+mod revert {
+
+    use super::*;
+
+    #[tokio::test]
+    #[should_panic(expected = "Revert(42)")]
+    async fn panics_after_claim_period() {
```

Comment:
Not a big deal but I would personally not prepend `panics_` so that the running tests say `revers::after_claim_period` as opposed to `reverts::panics_after_claim_period`

Reply:
I agree this reads better. I believe there are a few applications that follow this format so this should be updated.